### DISCUSSION TITLE
Safari can't open the page.

### DISCUSSION THREAD

**Discussion by procarrera** [vercel/vercel#4780] [Help] [answered]
Hi,
I have some projects hosted at Zeit/Vercel and i realized none of them is opening property in Safari/MacOS due to "bad secure connection", just like this:

![Download](https://user-images.githubusercontent.com/22551442/71076426-fbf92e80-2185-11ea-8675-e45715e49373.png)

**Would you have any solution for this?**

https://esther-bonder.vercel.app
https://share-me.now.sh
https://flordecaramelogourmet.com.br/

@TooTallNate

**Comment by TooTallNate** [comment, marked as answer]
Looks like you're using Safari for Windows, which was discontinued by Apple many years ago. Most likely it doesn't have compatibility with SSL certificates that use a modern cipher. That is to say, Safari for Windows should no longer be used on the modern web, I suggest using Firefox or Chrome.

**Reply by procarrera** [reply]
Hi !! unfortunately it s not the case, i am opening from OS X Yosemite 10.10.5

Would you need any specific info about system? i can't even open vercel.com here

![Captura de Tela 2020-07-03 às 22 34 06](https://user-images.githubusercontent.com/53456120/86502817-cc0a2500-bd7d-11ea-879e-526c3b69ab92.png)

**Reply by TooTallNate** [reply]
Well, again, Yosemite is rather old and [unsupported](https://en.wikipedia.org/wiki/OS_X_Yosemite#Release_history) by Apple at this point. It's possible that it has outdated SSL libraries. Did you manage to try Firefox or Chrome instead?

**Comment by SSylvain1989** [comment]
> Hi,
> I have some projects hosted at Zeit/Vercel and i realized none of them is opening property in Safari/MacOS due to "bad secure connection", just like this:
> 
> ![Download](https://user-images.githubusercontent.com/22551442/71076426-fbf92e80-2185-11ea-8675-e45715e49373.png)
> 
> **Would you have any solution for this?**
> 
> https://esther-bonder.vercel.app
> https://share-me.now.sh
> https://flordecaramelogourmet.com.br/
> 
> @TooTallNate

Hello did you find a solution ? 
Same issues here. 
Thanks

**Reply by markozxuu** [reply]
> > Hi,
> > I have some projects hosted at Zeit/Vercel and i realized none of them is opening property in Safari/MacOS due to "bad secure connection", just like this:
> > ![Download](https://user-images.githubusercontent.com/22551442/71076426-fbf92e80-2185-11ea-8675-e45715e49373.png)
> > **Would you have any solution for this?**
> > https://esther-bonder.vercel.app
> > https://share-me.now.sh
> > https://flordecaramelogourmet.com.br/
> > @TooTallNate
> 
> Hello did you find a solution ?
> Same issues here.
> Thanks

I don't understand the the reason for supporting an old browser. This is a statistic for the month of February 2021 and we can see that Chrome has a large adoption margin by the majority of users around the world

site: https://www.w3counter.com/globalstats.php

<img width="892" alt="Screen Shot 2021-03-11 at 18 15 07" src="https://user-images.githubusercontent.com/16585386/110872803-1ee59600-8296-11eb-9534-14f23d8e9ce6.png">

even in the list of the 10 most used browsers, your safari version does not appear
<img width="439" alt="Screen Shot 2021-03-11 at 18 18 54" src="https://user-images.githubusercontent.com/16585386/110872946-5e13e700-8296-11eb-86db-1c3e6d724733.png">

**Reply by SSylvain1989** [reply]
I will deploy on netlify hope it solve this issue.

**Reply by freearhey** [reply]
@SSylvain1989 did it help?

**Reply by SSylvain1989** [reply]
@freearhey yes . much better on netlify . But i was in difficulty whith vercel because i was using a custom font ( on nextJS ) ....

**Reply by leerob** [reply]
Unless I am mistaken, Netlify also uses modern SSL certificates. Could you point me to documentation stating otherwise?

**Reply by SSylvain1989** [reply]
@leerob hello :) 
if it's helpfull you can see here : https://github.com/vercel/next.js/issues/9851#issuecomment-805320351

**Reply by leerob** [reply]
That is referencing Safari 10 support with Next.js, which is the same regardless of the platform you deploy to.

**Reply by freearhey** [reply]
If anyone else encounters this problem, try to use a custom domain and Cloudflare as a proxy: https://vercel.com/support/articles/using-cloudflare-with-vercel#with-proxy 
In my case it worked and Safari accepted their certificates.
